src/backend/aspen/workflows/nextstrain_run/export.py

```diff
@@ -115,14 +115,12 @@ def export_run_config(
             session, county_samples, sequences_fh, metadata_fh
         )
 
-        # Write include.txt file(s) for targeted/non_contextualized builds.
-        if phylo_run.tree_type != TreeType.OVERVIEW:
-            selected_samples: List[PathogenGenome] = [
-                inp for inp in phylo_run.inputs if isinstance(inp, PathogenGenome)
-            ]
-            num_included_samples = write_includes_file(
-                session, phylo_run.gisaid_ids, selected_samples, selected_fh
-            )
+        selected_samples: List[PathogenGenome] = [
+            inp for inp in phylo_run.inputs if isinstance(inp, PathogenGenome)
+        ]
+        num_included_samples = write_includes_file(
+            session, phylo_run.gisaid_ids, selected_samples, selected_fh
+        )
```

Comment:
We no longer need to put this code in an `if` block, we'll check for selected samples for both scheduled and on-demand overview trees, but the `include.txt` file will just be empty for scheduled runs.